mycli/main.py

```diff
@@ -799,7 +823,10 @@ def format_output(self, title, cur, headers, expanded=False,
                 formatted = self.formatter.format_output(
                     rows, headers, format_name='vertical')
 
-            output.append(formatted)
+            if isinstance(formatted, (str, text_type)):
```

Comment:
This is a workaround for Python 2.7, without str pytest breaks at `test/test_main.py:133`